prov/rxm/src/rxm.h

```diff
@@ -66,6 +67,9 @@
 #define RXM_BUF_SIZE	16384
 
 #define RXM_SAR_LIMIT	262144
+#define RXM_SAR_TX_ERROR	UINT64_MAX
+#define RXM_SAR_RX_INIT		UINT64_MAX
```

Comment:
Is there a possibility for one of these macros to be misidentified as other?

Reply:
Also do we have asserts so that any actual msg_id is not `RXM_SAR_TX_ERROR`?

Reply:
I checked code and don't see places that can lead to the problem that the one of these macros to be misidentified as other, because they are used for different entries (RXM_SAR_TX_ERROR - only for TX entry, RXM_SAR_RX_INIT - only for RX entry)

Reply:
> Also do we have asserts so that any actual msg_id is not RXM_SAR_TX_ERROR?

`rxm_finish_sar_segment_send` handles CQ events for completed SAR segment TX requests. I don't see places where we can have `assert` to handle incorrect msg_id.
Btw, we reset msg_id in `rxm_ep_sar_tx_send`, it shouldn't be issue